ide/texmacspp.ml

```diff
@@ -382,7 +382,7 @@ and pp_token ?loc tok =
   let tokstr =
     match tok with
     | String s -> PCData s
-    | Numeral n -> PCData (to_string n) in
+    | Numeral (n,s) -> PCData (if s then n else "- "^n) in
```

Comment:
Shouldn't this be `"-"^n` for printing compatibility?  Also, you should probably strip leading zeros, no?